test_data/yast/autoyast/profiles/mini_s390x.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
this file looks repeated comparing with the one above.

Reply:
Since i have no idea how to make good name for this file, maybe mini_aarch64_s390x.yaml?